src/scenes/AcceleratorRouter/Overview/index.tsx

```diff
@@ -70,11 +72,12 @@ const OverviewView = () => {
   });
 
   return (
-    <Page title={strings.OVERVIEW}>
+    <AcceleratorMain>
+      <PageHeader title={strings.OVERVIEW} />
```

Comment:
Any chance we can get this to work with `Page` so we aren't depending more on other components? Is there some style we can pick up?